### DISCUSSION TITLE
How to give user access to specific workflow in a repository?

### DISCUSSION THREAD

**Discussion by Youssefguba** [codemagic-ci-cd/codemagic-docs#2587] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I am trying to give user permission to specific workflow not all workflows, but I can't

### Expected results

Expected to give a user access to a specific workflow not all workflows.

### Actual results

No specific access in workflows?

**Comment by dtrdic** [comment, marked as answer]
Hi @Youssefguba at the moment, we only support team-level permissions. This means that if a user is part of your team, they will be able to see all apps and workflows. However, we are currently in the process of implementing more structured roles and permissions. Unfortunately, I can't give you an ETA on that yet.

**Reply by envictor** [reply]
Hi @dtrdic, do you have an ETA by now?

**Reply by dtrdic** [reply]
@envictor not yet, sorry:/
